Bucketeer/Sources/Internal/Evaluation/EvaluationInteractor.swift

```diff
@@ -120,4 +149,49 @@ final class EvaluationInteractorImpl: EvaluationInteractor {
     func clearUpdateListeners() {
         updateListeners.removeAll()
     }
+
+    func saveFeatureTag(value: String) {
+        // https://github.com/bucketeer-io/android-client-sdk/issues/69
+        // 2- Save the featureTag in the SharedPreferences if it is configured in the BKTConfig
+        // 3- Clear the userEvaluationsID in the SharedPreferences if the featureTag changes
+        guard !value.isEmpty else {
```

Comment:
The condition must be the comparison of the current with the new tag.
The developer can configure the tag as empty if he needs.

Reference.
https://github.com/bucketeer-io/javascript-client-sdk/blob/main/src/internal/evaluation/EvaluationStorage.ts#L119-L131

```suggestion
        let featureTag = stateStorage.getStates().featureTag
        if value != featureTag {
            stateStorage.setFeatureTag(value: value)
            stateStorage.setCurrentEvaluationsId(value: "")
        }
```

Reply:
I got it, thank you for explaining

Reply:
fixed